src/plugins/links/public/components/editor/links_editor.tsx

```diff
@@ -67,16 +66,16 @@ const LinksEditor = ({
   onClose,
   initialLinks,
   initialLayout,
-  parentDashboard,
+  parentDashboardId,
   isByReference,
   flyoutId,
 }: {
-  onSaveToLibrary: (newLinks: Link[], newLayout: LinksLayoutType) => Promise<void>;
-  onAddToDashboard: (newLinks: Link[], newLayout: LinksLayoutType) => void;
+  onSaveToLibrary: (newLinks: ResolvedLink[], newLayout: LinksLayoutType) => Promise<void>;
+  onAddToDashboard: (newLinks: ResolvedLink[], newLayout: LinksLayoutType) => void;
   onClose: () => void;
-  initialLinks?: Link[];
+  initialLinks?: ResolvedLink[];
```

Comment:
Links aren't being updated properly:


https://github.com/elastic/kibana/assets/8698078/989e0a6e-9ec2-418b-9c92-2f634c841a7f

Reply:
🤦  fixed in https://github.com/elastic/kibana/pull/178670/commits/3305d783440519a28656f4abe133684186827ccf